commands/FBPrintCommands.py

```diff
@@ -424,3 +425,60 @@ def run(self, arguments, options):
       actionsDescription = fb.evaluateExpressionValue('(id)[{actions} componentsJoinedByString:@", "]'.format(actions=actions)).GetObjectDescription()
 
       print '{target}: {actions}'.format(target=targetDescription, actions=actionsDescription)
+
+class FBPrintAsCurl(fb.FBCommand):
+  def name(self):
+    return 'pcurl'
+
+  def description(self):
+    return 'Print the NSURLRequest (HTTP) as curl command.'
+
+  def options(self):
+    return [
+      fb.FBCommandArgument(short='-p', long='--portable', arg='portable', boolean=True, default=False, help='Embed request data as base64.'),
```

Comment:
Ah ok. Big amounts of data!

I still do think that `portable` is a bad name. Maybe even it should be `dataStyle` and then options are `tmp` or `base64` and it defaults to the one which makes most sense for the current platform.